calico_node/filesystem/etc/rc.local

```diff
@@ -8,37 +8,50 @@ if [ "$CALICO_DISABLE_FILE_LOGGING" == "true" ]; then
 	rm -r /etc/service/bird6/log
 	rm -r /etc/service/confd/log
 	rm -r /etc/service/felix/log
+	rm -r /etc/service/calico-bgp-daemon/log
 fi
 
-if [ "$CALICO_NETWORKING" == "false" ]; then
+case "$CALICO_NETWORKING_BACKEND" in
```

Comment:
I like the case statment, but can they be `==` instead of `in`? Makes it a bit more human-readable